src/libraries/System.Text.Encodings.Web/tools/updating-encodings.md

```diff
@@ -8,30 +8,30 @@ The current version of the Unicode data checked in is __12.1.0__. The archived f
 
 ```txt
 dotnet run -- "path_to_UnicodeData.txt" ../../src/System/Text/Unicode/UnicodeHelpers.generated.cs
 ```
 
-3. Open a command prompt and navigate to the __src/libraries/System.Text.Encodings.Web/tools/GenUnicodeRanges__ directory, then run the following command, replacing the first parameter with the path to the _Blocks.txt_ file you downloaded in the first step. This command will update the `UnicodeRanges` type in the runtime folder and update the unit tests to exercise the new APIs.
+4. Open a command prompt and navigate to the __src/libraries/System.Text.Encodings.Web/tools/GenUnicodeRanges__ directory, then run the following command, replacing the first parameter with the path to the _Blocks.txt_ file you downloaded earlier. This command will update the `UnicodeRanges` type in the runtime folder and update the unit tests to exercise the new APIs.
 
 ```txt
 dotnet run -- "path_to_Blocks.txt" ../../src/System/Text/Unicode/UnicodeRanges.generated.cs ../../tests/UnicodeRangesTests.generated.cs
 ```
 
-4. Update the __ref__ APIs to reflect any new `UnicodeRanges` static properties which were added in the previous step, otherwise the unit test project will not be able to reference them. See https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/updating-ref-source.md for instructions on how to update the reference assemblies.
-
-5. Update the __src/libraries/System.Text.Encodings.Web/tests/System.Text.Encodings.Web.Tests.csproj__ file to reference the new file that you dropped into the __src/libraries/Common/tests/data__ folder in the first step above. Open the .csproj file in a text editor, and replace both the `<EmbeddedResource>` and the `<LogicalName>` elements at the end of the document to reference the new file name.
+5. Update the __ref__ APIs to reflect any new `UnicodeRanges` static properties which were added in the previous step, otherwise the unit test project will not be able to reference them. See https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/updating-ref-source.md for instructions on how to update the reference assemblies.
 
-6. In the file __src/libraries/System.Text.Encodings.Web/tests/UnicodeHelpersTests.cs__, update the _UnicodeDataFileName_ const string at the top of the file to reflect the new file name for _UnicodeData.x.y.txt_.
+6. Update the __src/libraries/System.Text.Encodings.Web/tests/System.Text.Encodings.Web.Tests.csproj__ file to reference the new __UnicodeData.txt__ file that was added to the _runtime-assets_ repo in step (1). Open the .csproj file in a text editor and replace the `<UnicodeUcdVersion>` element near the top of the file to reference the new UCD version being consumed.
```

Comment:
`Element` -> property. 
Add a link to the runtime-assets repo subtree with the test data.
```suggestion
6. Update the __src/libraries/System.Text.Encodings.Web/tests/System.Text.Encodings.Web.Tests.csproj__ file to reference the new __UnicodeData.txt__ file that was added to the [runtime-assets](https://github.com/dotnet/runtime-assets/tree/master/src/System.Private.Runtime.UnicodeData) repo in step (1). Open the .csproj file in a text editor and replace the `<UnicodeUcdVersion>` property near the top of the file to reference the new UCD version being consumed.
```